Samples/7.ServerList/Program.cs

```diff
@@ -45,15 +43,17 @@ static void Main( string[] args )
                 if ( !uint.TryParse( File.ReadAllText( "cellid.txt"), out cellid ) )
                 {
                     Console.WriteLine( "Error parsing cellid from cellid.txt. Continuing with cellid 0." );
+                    cellid = 0;
```

Comment:
`cellid` will be `default(uint)` in this case which will be `0` anyway, but it might be good to explicitly state this for code readability.

Reply:
Actually it’ll be whatever `uint.TryParse` sets it to, which is documented as “zero if the conversion failed”, but either way I just added it in for readability.